### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by shzeshzeshze** [input-output-hk/react-polymorph#185]
I'm coming from the [Daedalus](https://github.com/input-output-hk/daedalus) repo and yesterday [I posted an issue there](https://github.com/input-output-hk/daedalus/issues/2575) concerning it's accessibility. After forking it and running mainnet locally I see most of the UI is based of react-polymorph.

What are the team's thoughts on adopting and implementing some sort of accessibility standard? As a recommendation I would suggest [WCAG 2.1](https://www.w3.org/TR/WCAG21/) at AA conformance level. This [WCAG quick reference guide](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa) is a great way to find all the different success criterion, why each is important, and success and failure modes for each.

I have not had time to evaluate each component individually but one that stuck out in my mind were the Select components (in react-polymorph) that make up the ProfileSettings component (in Daedalus). All of the labels are hardcoded with `aria-hidden="true"` and `role="presentation"` when in fact they are quite important to screen reader users.

I really like both of these projects so far and would love to see improved accessibility so that the UI in the applications IOHK puts out provide an equivalent experience to the widest possible audiences.

Thanks for your time.
